JsonLd/EventListener/ResourceContextBuilderListener.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\JsonLd\EventListener;
+
+use Dunglas\ApiBundle\JsonLd\Event\ContextBuilderEvent;
+use Dunglas\ApiBundle\JsonLd\Event\Events;
+use Dunglas\ApiBundle\Mapping\ClassMetadataFactoryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Serializer\NameConverter\NameConverterInterface;
+
+/**
+ * Build default context for Json-Ld resources.
```

Comment:
`JSON-LD` (just the case). Also, phrase must be at the 3rd person in the PHPDoc (`Builds`).